pkg/controller/case_test.go

```diff
@@ -1,3 +1,5 @@
+// +build integration
+
```

Comment:
are these actually integration tests, or can they be moved with all of the others that already exist?

Reply:
> are these actually integration tests,

those tests have the same contracts as the current integration tests but just not starting the API Server.


Some time ago we discussed that current integration tests will not work for CRDs implementation and because of that, there was created such issue: https://github.com/kubernetes-sigs/service-catalog/issues/2597

basically, the goal is to replace the integration tests of the API Server with the logic which does not require to start the API Server at all. Thanks to that we still testing the contract described in integration tests but such approach allows us to do that both for API Server and CRD implementation.

The first part was already migrated: https://github.com/kubernetes-sigs/service-catalog/pull/2580#issue-260251250

This is the second PR, and after merging that we will submit the 3rd PR - the final one which will move all rest tests (we have that PR already prepared)

about adding `// +build integration`  tag, please ready [this](https://github.com/kubernetes-sigs/service-catalog/pull/2596#issuecomment-501650071) comment